db/main/migrate/20190820082431_add_vcs_id_to_organization.rb

```diff
@@ -0,0 +1,15 @@
+class AddVcsIdToOrganization < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def up
+    add_column :organizations, :vcs_id, :string, default: nil
+
+    add_index :organizations, %i[vcs_id vcs_type], unique: true
```

Comment:
@weban, It's better to use `CREATE INDEX CONCURRENTLY ...`, otherwise it may block the table. See [#184](https://github.com/travis-ci/travis-migrations/pull/184/files#diff-2885d69eeb69c538d83d9e7f8bfac5b9R5) for example

Also, will unique constraint work here? We create an empty `vcs_id` right before that, which means that all records will have `vcs_type = 'GithubRepository'` and `vcs_id = nil`